Lib/ufo2ft/__init__.py

```diff
@@ -30,14 +29,20 @@ def compileOTF(ufo, outlineCompilerClass=OutlineOTFCompiler,
       By default, all floats are rounded to integer (tolerance 0.5); a value
       of 0 completely disables rounding; values in between only round floats
       which are close to their integral part within the tolerated range.
+
+    *featureWriters* argument is a tuple of FeatureWriter instances.
+    Features will be written by each FeatureWriter in the given order
     """
 
     outlineCompiler = outlineCompilerClass(
         ufo, glyphOrder, roundTolerance=roundTolerance)
     otf = outlineCompiler.compile()
 
+    if featureWriters is None:
+        featureWriters = (KernFeatureWriter(ufo), MarkFeatureWriter(ufo))
```

Comment:
defaultFeatureWriters should be a top-level export imo so that clients can take the default list and add/remove to/from it.

Reply:
do so would avoid duplicating it inside ufo2ft as well.

Reply:
Is a function `defaultFeatureWriter(font)` that returns the list of feature writer instances OK?

Reply:
Why not a list of classes?